history-local-graph-db.md

```
@@ -0,0 +1,1203 @@
+  → `CALL db.idx.vector.queryNodes('Doc','embedding',k, vecf32([...])) YIELD node, score`
+  returned correct cosine ordering (exact match score 0.0).
+- **Relationship** index + KNN on `r.fact_embedding` (our `:RELATES_TO` claim
+  edge): `CREATE VECTOR INDEX FOR ()-[r:RELATES_TO]-() ON (r.fact_embedding) OPTIONS {...}`
+  → `CALL db.idx.vector.queryRelationships('RELATES_TO','fact_embedding',k, vecf32([...]))`.
+- Inline `vec.cosineDistance(vecf32([...]), vecf32([...]))` function (no index).
+- `db.indexes()` lists both VECTOR indexes `OPERATIONAL` (HNSW: M=16,
+  efConstruction=200, efRuntime=10).
+
+**Correction to Phase 0:** the earlier "vector index FAILED" result was a
+*syntax* mismatch, not a capability gap — Phase 0 used Neo4j's
+`OPTIONS { indexConfig: { `vector.dimensions`: … } }` DDL. FalkorDB's **native**
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)